docs/src/charm/howto/index.md

```diff
@@ -16,6 +16,7 @@ Overview <self>
 
 charm
 install-lxd
+openstack
```

Comment:
I imagine at some point, we're going to need to make an "Integrate with Clouds", and detail other clouds like aws, azure, gcp, maas, etc.  Even lxd could be counted as one of these clouds (though there isn't much to integrate with)